Makefile

```diff
@@ -146,7 +146,8 @@ $(eval $(call build-package,pcre2,10.42-r0))
 $(eval $(call build-package,git,2.39.0-r1))
 $(eval $(call build-package,bash,5.2_rc4-r0))
 $(eval $(call build-package,go-stage0,1.19.1-r0))
-$(eval $(call build-package,go,1.19.4-r1))
+$(eval $(call build-package,go-1.18,1.18.9-r0))
```

Comment:
do we need to build this one? 1.18.10 is out and 1.19.5 as well

Reply:
1.19.5 will be added in another PR.

We won't build any old packages as they're no supported upstream, but the packages for previously built versions are still available (e.g., 1.19.3, 1.19.1)

Reply:
thanks